app/Http/Controllers/CommentController.php

```diff
@@ -110,7 +110,7 @@ public function store($postId)
         ]);
 
         if (CommentRepository::validate($input, array_keys($input))->fails()) {
-            throw new BadRequestHttpException('Your comment was empty.');
```

Comment:
Well man, sorry, I was very motivated those days and with a lot of free time, but, I'm kinda busy right now. I'm sure that you had been busy and that is why you take all this time to answer. I'll understand if you wanna dismiss my pull requests.